### DISCUSSION TITLE
Accessing multiple ControlTheme using Style Selector

### DISCUSSION THREAD

**Discussion by LilyStilson** [AvaloniaUI/Avalonia#9010] [Q&A]
As it turns out, the [`<Expander />`](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml) control has multiple `<ControlTheme />` with different Keys in both its default and Fluent styles.

Is it possible to access these `<ControlTheme />` using `<Style Selector="" />`? And if so, then how to properly do it?

**Comment by maxkatz6** [comment]
No. Why?
You still can access Expander with `Selector="Expander"` as it was possible before. It's not possible to modify existing control theme, as it was one of problems control theme solves. Instead, you can use BaseOn to modify them.

> the [<Expander />](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml) control has multiple <ControlTheme /> with different Keys in both its default and Fluent styles

Expander itself has only one ControlTheme per themes package. Everything else - internal themes for the toggle button.
